pytorch_lightning/utilities/cli.py

```diff
@@ -344,7 +370,7 @@ def instantiate_trainer(self) -> None:
         callbacks = [self.config_init[c] for c in self.parser.callback_keys]
         self.config_init['trainer']['callbacks'].extend(callbacks)
         if 'callbacks' in self.trainer_defaults:
-            if isinstance(self.trainer_defaults['callbacks'], list):
+            if inspect.isclass(self.trainer_defaults['callbacks']):
```

Comment:
This is wrong. Checking that it is a list is not the same as checking if it is a class.

Reply:
Whoops just an accidental bug, I didn't think those two checks were equivalent haha. Fixed now